### ISSUE TITLE
Add ability to record Jitsi conference

### ISSUE THREAD

**Issue by akontsevich** [vector-im/riot-web#5925]
### Description
Please add ability to record conference in Jitsi plugin

### Version information
- **Platform**: web (in-browser) and desktop
- **OS**: Windows, macOS, Ubuntu, Linux, etc

**Comment by MurzNN** [comment]
At now you can use something like https://www.webrtc-experiment.com/RecordRTC/ to record conference in web browser.

**Comment by akontsevich** [comment]
@MurzNN, does it work only in browser version, not desktop? Will it work with Jitsi?

**Comment by MurzNN** [comment]
@akontsevich it was work for me in browser (if I open Riot in browser), and, as I remember, it works normally with Jitsi conference too. You can try it yourself online on that site, without installing.

**Comment by akontsevich** [comment]
@MurzNN, records nothing for me for now in Jitsi.

**Comment by MurzNN** [comment]
Seems Jitsi widgets allow Live streaming & Recording on YouTube: https://jitsi.org/live-streaming-and-recording-a-jitsi-conference/ but seems this feature is disabled in http://jitsi.riot.im/ installation. So seems we can enable Riot conference recording via installing separate Jitsi instance on homeserver, or ask http://jitsi.riot.im/ owners to turn on this feature.

**Comment by akontsevich** [comment]
Yes adding recording and Youtube live stream features to Jitsi integration plugin for Riot would be great!!! Killing feature for Skype replacement! With screensharing as well of course: #4880. Waiting for implementation. 

Also good to have ability to unfold Jitsi widget to whole chat area or even whole Riot window! Will create ticket for this.

**Comment by hex-m** [comment]
As a server operator who is able to set up a Jitsi Meet Instance for their homeserver this is already possible with Jibri as mentioned above. Such a setup was used for FOSDEM 2021 and you can read about it [here](https://matrix.org/blog/2021/02/15/how-we-hosted-fosdem-2021-on-matrix).

The plan seems to be to implement Matrix-native group calling instead of the Jitsi-Meet widget.
